profiles/lib/traffic_signal.lua

```diff
@@ -0,0 +1,25 @@
+local traffic_lights = traffic_lights
```

Comment:
Nit. I may be wrong, but it seems this is redundant(but probably I would somehow highlight that this enum declared from outside in C++ code)

Reply:
I've no idea where that's come from, will remove.